design/ceph/object/ceph-sse-s3.md

```diff
@@ -0,0 +1,42 @@
+---
+# AWS server side encryption SSE-S3 support for RGW
+target-version: release-1.10
+---
+
+# Feature Name
+AWS server side encryption SSE-S3 support for RGW
+
+## Summary
+The S3 protocol supports three different types of [server side encryption](https://docs.aws.amazon.com/AmazonS3/latest/userguide/serv-side-encryption.html): SSE-C, SSE-KMS and SSE-S3. For the last two RGW server need to configure with external services such as [vault](https://www.vaultproject.io/). Currently Rook configure RGW with `SSE-KMS` options from v1.6 which needs atleast  cephv15, so that S3 request with `sse:kms` header will be handled properly. Recently the support for handling the `sse:s3` added to RGW(planning to support from cephv17 onwards) with another set of options, so Rook needs to provide option to configure RGW similar to `SSE-KMS`.
```

Comment:
This feature is only for quincy, right? 
```suggestion
The S3 protocol supports three different types of [server side encryption](https://docs.aws.amazon.com/AmazonS3/latest/userguide/serv-side-encryption.html): SSE-C, SSE-KMS and SSE-S3. For the last two RGW server need to configure with external services such as [vault](https://www.vaultproject.io/). Currently Rook configure RGW with `SSE-KMS` options to handle the S3 requests with the `sse:kms` header. Recently the support for handling the `sse:s3` was added to RGW, so Rook will now provide the option to configure RGW with `sse:s3`.

The `sse:s3` is supported only from Ceph v17 an onwards, so this feature can only be enabled for Quincy or newer.
```

Reply:
Yaa the feature will be available in Quincy